libs/perun/services/src/lib/table-checkbox.service.ts

```diff
@@ -17,26 +17,38 @@ export class TableCheckbox {
   private itemsCheckedCounter: number;
 
   // checks if all rendered rows are selected (in this function also disabled checkboxes are allowed)
-  isAllSelectedWithDisabledCheckbox<T>(
+  isAllSelected<T>(
     rowsSelected: number,
-    filter: string,
-    pageSize: number,
-    nextPage: boolean,
-    pageIndex: number,
     dataSource: MatTableDataSource<T>,
-    sort: MatSort,
-    canBeSelected: (T) => boolean
+    canBeSelected?: (T) => boolean
```

Comment:
If we provide a default select function that just returns true, there is no need for the `if(!canBeSelected)` block I believe.
```suggestion
    canBeSelected: (item: T) => boolean = (): boolean => true
```